firmware/thruna/DCM/Inc/App/App_Regen.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "App_CanRx.h"
+#include "App_CanTx.h"
+#include "App_SharedMacros.h"
+
+#define DEG_TO_RAD(degrees) ((degrees) * (float)M_PI / 180.0f)
+#define WHEELBASE_mm                   1550
+#define TRACK_WIDTH_mm                 1100
+#define APPROX_STEERING_TO_WHEEL_ANGLE 0.3f
+#define MAX_TORQUE_CHANGE              10.0f
+#define MIN_SCALING_SPEED_kph          35.0f
+#define SPEED_MIN_kph                  5.0f
+#define MAX_PEDAL_POSITION             -100.0f
```

Comment:
Why -100?

Reply:
the range of the pedal mapping changes from [0,100] to [-100,100] I mentioned it in compute_regen_torque_request but I can add a comment for this somewhere